src/tests/stress/pedsim/Ped_util/Global_funs.enc

```diff
@@ -0,0 +1,315 @@
+-- This file was automatically converted by encorec
+
+module Global_funs
+import Agent_passive
+import XML
+fun invSqrt(input : int) : real
+  var ret = 0.0
+  EMBED (unit)
+    {
+  float x = #{input};
+  float xhalf = 0.5f * x;
+  int i = *(int*)&x;
+  i = 0x5f3759df - (i >> 1);
+  x = *(float*)&i;
+  #{ret}= x*(1.5f - xhalf*x*x);
+  }
+  END
+  ret
+end
+fun find_extreme(agents : [Agent]) : (int, int, int, int)
+  var extreme = (-(10000), -(10000), 10000, 10000)
+  var i = 0
+  for a <- agents do
+    i = i + 1
+    extreme = extreme_check(a.pos(), extreme)
+    for index <- [0..|a.targets_x| - 1] do
+      extreme = extreme_check(((a.targets_x)(index), (a.targets_y)(index)), extreme)
+    end
+  end
+  match extreme with
+    case (xmax, ymax, xmin, ymin) =>
+      (xmax + 10, ymax + 10, xmin - 10, ymin - 10)
+    end
+
+  end
+end
+fun extreme_check(a : (int, int), b : (int, int, int, int)) : (int, int, int, int)
```

Comment:
you should try to be consistent when naming things, e.g. `fun invSqrt(input : int) : real` vs `fun extreme_check(a : (int, int), b : (int, int, int, int)) : (int, int, int, int)`. Choose one, both do not make sense (camel case or snake case)